### ISSUE TITLE
Meeting list add to calendar tabindex

### ISSUE THREAD

**Issue by kevindherman** [proudcity/wp-proudcity#2111]
**Source**
Marin Watershed https://marin-watershed-program-ca.proudcity.com/

**Feedback**
The homepage includes 2 links (“Add to calendar”) that have a positive tabindex (https://dequeuniversity.com/tips/tabindex-positive-numbers) which means they are, incorrectly, first in the tab order.

**Comment by curtismchale** [comment]
This file is `wp-proud-theme/templates/teaser-items/teaser-events-list.php` at line `53` is where the link shows up but when I set the `tabindex` in the HTML (which is currently blank) it does nothing to affect the main page display. So that means our `tabindex` is being set in some other spot that is not currently identified.

**Comment by curtismchale** [comment]
Actually it's the addtocalendar.js stuff in the bottom of `wp-proud-theme/templates/teaser-items/teaser-meting-list.php` on line `54`. Including that JS on the page forced the tabindex.

**Comment by curtismchale** [comment]
@kevindherman so the "old" way of doing the add to calendar buttons is just going to mess with our `tabindex` values. I looked at the service we were using and it [now is a paid service](https://addtocalendar.com/).

Unless I hear differently, I'm going to use this [newer script](https://add-to-calendar-button.com/) to build our buttons. It will result in a bit different styling for the buttons but the functionality should be the same.

**Comment by curtismchale** [comment]
@kevindherman @lukefretwell this is what the new add to calendar button will look like.

<img width="463" alt="CleanShot 2022-10-12 at 11 47 44@2x" src="https://user-images.githubusercontent.com/74561/195423842-f69ed73d-0c30-4ec0-b1cc-383295d9e430.png">

**Comment by curtismchale** [comment]
@kevindherman for QA we'll need to check all the Add to Calendar buttons for meetings and events as I upgraded them to the new button so they remain uniform and don't have any `tabindex` set from the old script.

@lukefretwell

**Comment by kevindherman** [comment]
@curtismchale I'm fine with the design aspect. @lukefretwell thoughts?

For testing, am I just looking for the dropdown opening and directing to those options? I'm not sure what exactly tabindex actually is, so that aspect needs testing, let me know what I should be looking for. 

List of event views with add to calendar feature to test:

- [ ] event list (list view)
- [ ] event list (mini list view)
- [ ] event detail
- [ ] calendar
- [ ] meeting list (list view)
- [ ] meeting detail

**Comment by kevindherman** [comment]
@curtismchale we got a ticket that appears to be an outlier where the new calendar icon is huge using the list view:

https://www.lucastexas.us/
![Homepage](https://user-images.githubusercontent.com/6032426/197556637-64548a37-a849-4534-be25-8e79c9a9ebfb.jpeg)

Here's Williams County using the same widget and it displays as intended: https://www.williamsnd.com/calendar-list-view/

**Comment by kevindherman** [comment]
I am able to replicate this on Lucas as shown above. However, I just got a ticket from Holyoke and I can replicate it in Firefox but it's slightly different in Chrome. 

Firefox: https://www.holyoke.org/events
<img width="1172" alt="Screen Shot 2022-10-24 at 1 11 50 PM" src="https://user-images.githubusercontent.com/6032426/197585565-8622e5da-da2b-46ee-8848-cd1c6849b4c5.png">

Chrome: https://www.holyoke.org/events
![Screen Shot 2022-10-24 at 1 12 22 PM](https://user-images.githubusercontent.com/6032426/197585663-78b1d871-ae0d-4a65-8ed5-9ffae44d37d7.png)

Those are both events, for meetings content it's a little different:
Meeting in firefox: https://www.holyoke.org/meetings/charter-and-rules-committee-meeting-october-24-2022/
<img width="1134" alt="Screen Shot 2022-10-24 at 1 13 26 PM" src="https://user-images.githubusercontent.com/6032426/197585907-42fca55c-6a11-4819-a284-9f5d32f84f33.png">

Meeting in chrome: https://www.holyoke.org/meetings/charter-and-rules-committee-meeting-october-24-2022/
![Screen Shot 2022-10-24 at 1 14 26 PM](https://user-images.githubusercontent.com/6032426/197586021-1e2142af-4d76-4f61-b5dc-d6919b3519fe.png)
